### PR TITLE
UNR-638 - Starter Project wasn't working out of the box because of missing folder

### PR DESCRIPTION
**Pull request by girayimprobable**
This folder is required for generating project files.

### CODE REVIEW

File: Game/Saved/Logs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```
Comment (danielimprobable) [inline Game/Saved/Logs/.gitignore]: Why don't we edit the main git ignore and get rid of this one?

Comment (girayimprobable) [reply Game/Saved/Logs/.gitignore]: Oh that makes sense I think :).

Comment (davedolben) [reply Game/Saved/Logs/.gitignore]: I think the point of this file was to make sure there was _something_ in this directory so the directory gets created on disk, right?

Comment (danielimprobable) [reply Game/Saved/Logs/.gitignore]: It sounds wrong to have to put in a file to this directory. I'm pretty sure unreal generates this folder when running. Do we have more details on what step in the setup process that generates this file. It shouldn't be required.

Comment (girayimprobable) [reply Game/Saved/Logs/.gitignore]: Right clicking uproject and clicking generate project simply does not work on a fresh clone.

**Pull request merged**